buildbot-conf/cordova.conf

```diff
@@ -0,0 +1,326 @@
+
+    # clone all repos
+    MedicClone(),
+
+    # install tools
+    NPMInstall(what='cordova-coho',        workdir='cordova-coho'),
+    NPMInstall(what='cordova-lib',         workdir='cordova-lib/cordova-lib'),
+    NPMInstall(what='cordova-cli',         workdir='cordova-cli'),
+    NPMInstall(what='cordova-js',          workdir='cordova-js'),
+    NPMInstall(what='cordova-plugman',     workdir='cordova-plugman'),
+    NPMInstall(what='platform',            workdir=I('cordova-%(prop:platform)s')),
+    NPMInstall(what='cordova-mobile-spec', workdir='cordova-mobile-spec/createmobilespec'),
+
+    # link the installed code
+    SH(command=['cordova-coho/coho', 'npm-link'], description='coho link'),
+
+    # prepare the test app
+    SH(
+        command = [
+            'node',
+            'cordova-mobile-spec/createmobilespec/createmobilespec.js',
+            I('--%(prop:platform)s')
+        ],
+        description='creating mobilespec app'
+    ),
+    SH(
+        command=[
+            'node',
+            'cordova-medic/updateconfig.js',
+            I('--%(prop:platform)s'),
+        ],
+        description='preparing mobilespec app'
+    ),
+]
+
+cordova_plugins_all = BuildFactory()
+cordova_plugins_all.addSteps(common_setup_steps)
+cordova_plugins_all.addSteps(common_plugins_steps)
+cordova_plugins_all.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js')], description='running tests'),
+])
+
+# WORKAROUND:
+#            this is here to match what medic already does; these
+#            should be their own builders, using a proper test matrix
+cordova_plugins_windows = BuildFactory()
+cordova_plugins_windows.addSteps(common_setup_steps)
+cordova_plugins_windows.addSteps(common_plugins_steps)
+cordova_plugins_windows.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store80"], description='running tests (Windows 8.0)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store"], description='running tests (Windows 8.1)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--phone"], description='running tests (Windows Phone 8.1)'),
+])
+
+c['builders'].extend([
+
+    # plugins builders
+    BuilderConfig(name='cordova_plugins_ios_osx',            slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'ios',        'slaveos': OSX}),
+    BuilderConfig(name='cordova_plugins_android_osx',        slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'android',    'slaveos': OSX}),
+    BuilderConfig(name='cordova_plugins_windows_windows',    slavenames=WINDOWS_SLAVES, factory=cordova_plugins_windows, properties={'platform': 'windows',    'slaveos': WINDOWS}),
+    BuilderConfig(name='cordova_plugins_wp8_windows',        slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'wp8',        'slaveos': WINDOWS}),
```

Comment:
This patch prevents device screen to be automatically locked. You can't install the app on the device if its screen is locked and there is no way to disable this (max timeout is a few minutes, not sure if there is something new here)